frontend/src/app/patients/EditPatient.tsx

```diff
@@ -317,15 +325,17 @@ const EditPatient = (props: Props) => {
           <Button
             id="edit-patient-save-upper"
             className="prime-save-patient-changes-start-test"
-            disabled={loading || !formChanged}
+            disabled={loading}
             onClick={() => {
               onSave(true);
             }}
             variant="outline"
             label={
               loading
                 ? `${t("common.button.saving")}...`
-                : "Save and start test"
+                : formChanged
```

Comment:
a double ternary like this reads a bit awkward, so it might be good to pull this into a variable in the component body like so 

```
const saveButtonLabel = formChanged ? "Save and start test" : "Start test" 
```
which you can use instead of the second level of nesting here.

Reply:
Ohhh TIL!  Agreed that makes much more sense than a nested ternary. 

However, setting it as a variable that isn't a function causes an error as `formChanged` is out of scope. So instead of a raw expression I set the var to a function where `formChanged` is passed as the argument (which I thought was pretty slick 💅 ).